### PR TITLE
LPD-33439 Display template page implementation for orders

### PR DESCRIPTION
**Pull request by alessiorendina**
https://liferay.atlassian.net/browse/LPD-33439

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:playwright

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:playwright

**Comment by alessiorendina** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/CommerceOrderHttpHelperImpl.java [hunk 1]
```diff
@@ -504,7 +504,7 @@ public void setCurrentCommerceOrder(
 			commerceOrder.getCommerceOrderId(),
 			_getCommerceContext(httpServletRequest));
 
-		if (!themeDisplay.isSignedIn()) {
+		if ((themeDisplay != null) && !themeDisplay.isSignedIn()) {
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/CommerceOrderHttpHelperImpl.java:507]: we need to open a bug for this I believe

**Review by andrea-ale-sbarra — APPROVED** [review]
Code looks good, just make sure to have the bug opened for the guest scenario.

Should we also open a ticket for making a new widget for the order detail?

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**